util/logs/workflow-logger.go

```diff
@@ -69,7 +69,11 @@ func WorkflowLogs(ctx context.Context, wfClient versioned.Interface, kubeClient
 	if logOptions == nil {
 		logOptions = &corev1.PodLogOptions{}
 	}
-	logOptions.Timestamps = true
+	logCtx.WithField("options", logOptions).Debug("Log options")
+
+	// make a copy of requested log options and set timestamps to true, so they can be parsed out later
+	podLogStreamOptions := *logOptions
```

Comment:
I think you could just use `logOptions` here

Reply:
Thats exactly what the problem was, the `logOptions` is a pointer. So when you set timestamps to true in the next line, you lose the original value which was set by user. This is important because line 115 uses the original `logOptions` set by user. My fix makes a copy of users `logOptions` and sets timestamps to true on the copy only, without altering the original value.